files/dbconfig.xml.epp

```diff
@@ -0,0 +1,68 @@
+<%# This template file needs to be in the files folder -%>
+<%# because we use the file() function to read the content -%>
+<%# as input to Deffered('inline_epp', ...) -%>
```

Comment:
Explain why these template files are in the files folder.

Reply:
reverted and moved files back to templates folder, then using `stdlib::deferrable_epp`